nestkernel/event.h

```diff
@@ -952,6 +952,12 @@ class GapJunctionEvent : public SecondaryEvent
     supported_syn_ids_.push_back( synid );
   }
 
+  static void
+  set_coeff_length( const size_t coeff_length )
+  {
+    coeff_length_ = coeff_length;
```

Comment:
I gave it another thought:

Something like
```cpp
static void
set_coeff_length( const size_t coeff_length )
{
  #pragma omp critical
  coeff_length_ = coeff_length;
}
```
would not solve this problem: There is no parallel region within this function. Also checking the number of threads in this function is no option (this would always be 1).

One just needs to make sure that this function is never called from several threads at the same time. As the `check_wfr_use` function is in a omp single region this is true here. So from my point of view there is nothing we could do here, do you agree?